okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -118,8 +118,8 @@ import kotlin.DeprecationLevel.ERROR
  * OkHttp also uses daemon threads for HTTP/2 connections. These will exit automatically if they
  * remain idle.
  */
-open class OkHttpClient internal constructor(
-  builder: Builder
+open class OkHttpClient internal @JvmOverloads constructor(
```

Comment:
Doesn't this make `new OkHttpClient()` an internal call?  That is a really common pattern for java clients.

Reply:
You're right. Interestingly, it looks like this doesn't change the public API for Java consumers, which can still use that constructor but does change the public API for Kotlin consumers. Perhaps the name mangling used to prevent access to internal members isn't applied to constructors.

If this is true, it means that all constructors that were originally private or package-private and become `internal` as part of the migration, became effectively public to Java consumers.